assets/js/homepage.js

```diff
@@ -18,8 +18,9 @@
 			return;
 		}
 
-		for ( var i = 0; i < homepageContent.querySelectorAll( '.entry-title, .entry-content' ).length; i++ ) {
-			homepageContent.querySelectorAll( '.entry-title, .entry-content' )[ i ].classList.add( 'loaded' );
+		var entries = homepageContent.querySelectorAll( '.entry-title, .entry-content' );
+		for ( var i = 0, l = entries.length; i < l; i++ ) {
```

Comment:
Being honest, I don't think there is a big performance gain assigning `entries.length` to `l`, but it does make the code a bit more verbose. Could we simplify it as so?
```suggestion
		for ( var i = 0; i < entries.length; i++ ) {
```

Reply:
I agree - also this is unusual style (multiple assign in the `for` statement). More idiomatic and no downside to just using `entries.length` directly as suggested above. Can you make this change please and then we'll merge :)

Reply:
Sounds good, done 👌

Reply:
Thank you! Comment when you're done testing and I'll merge :)

Reply:
@haszari is that a task for me or @Aljullu?

Reply:
Just confirming you've tested locally @valdrinkoshi :)